imageroot/templates/email.yaml

```diff
@@ -0,0 +1,42 @@
+type: email           # Don't change
```

Comment:
This file is quite simple, I rather use a bash script to generate it instead of a template.
Please do not change, it's just a personal preference.

Reply:
for now yes but tomorrow when/if we have an UI we could have more vars to change